hatch/src/repo.rs

```diff
@@ -14,8 +14,8 @@ pub fn hatchfile_path(base: &Path) -> PathBuf {
   base.join("Hatch.yml")
 }
 
-pub fn clone_dep(repo: &(String, String), path: &Path) {
-  Repository::clone(&repo.0, path.join(&repo.1));
+pub fn clone_dep(url: &str, name: &str, path: &Path) {
```

Comment:
Should this function just take a single path with name joined before hand?

Reply:
That would be intuitive. But I don't care that much.